en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md

```diff
@@ -0,0 +1,181 @@
+---
+id: network-cisco-firepower-snmp
+title: Cisco Firepower SNMP
+---
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+Cisco Firepower including CPU, Faults, Hardware, Interfaces, Memory.
```

Comment:
Maybe we can give more insights on what Firepower, FX OS is the embedded system or a function?

Reply:
it's the OS of the firewall. but i don't think we need to mention it.